DataManagementSystem/Client/FTS3Job.py

```diff
@@ -141,8 +133,7 @@ def __fetchSpaceToken( seName ):
     if seName:
       seObj = StorageElement( seName )
 
-
-      res = seObj.getStorageParameters( "SRM2" )
+      res = seObj.getStorageParameters("GFAL2_SRM2")
```

Comment:
I suggest you rather use the protocol parameter
`seObj.getStorageParameters(protocol = 'srm')`
Like this it is plugin independant